src/main/java/com/github/bartimaeusnek/ASM/CropStickTransformer.java

```diff
@@ -108,7 +107,9 @@
                 case 1: {
                     CropLoadCoreASM.cppASMlogger.info("Patching WeedingTrowel to accept custom Weeds");
                     String name = "onItemUseFirst";
-                    String desc = !isObfuscated ? "(Lnet/minecraft/item/ItemStack;Lnet/minecraft/entity/player/EntityPlayer;Lnet/minecraft/world/World;IIIIFFF)Z" : "(Ladd;Lyz;Lahb;IIIIFFF)Z";
+                    //String desc = !isObfuscated ? "(Lnet/minecraft/item/ItemStack;Lnet/minecraft/entity/player/EntityPlayer;Lnet/minecraft/world/World;IIIIFFF)Z" : "(Ladd;Lyz;Lahb;IIIIFFF)Z";
```

Comment:
This is the actual real change, without this commented out, the whole thing blows up IC2 (apparently because it replaces a deobfuscated name with an obfuscated one)